test/recipe_box_web/controllers/page_controller_test.exs

```diff
@@ -3,6 +3,6 @@ defmodule RecipeBoxWeb.PageControllerTest do
 
   test "GET /", %{conn: conn} do
     conn = get(conn, "/")
-    assert html_response(conn, 200) =~ "Welcome to Phoenix!"
+    assert html_response(conn, 200) =~ "Welcome to RecipeBox!"
```

Comment:
Just for my own understanding, is this test looking for the text "Welcome to RecipeBox!" anywhere in the HTML response?

Reply:
That's how I'm understanding it. My best guess is that it's similar to a RSpec/Capybara type thing, `expect(page.response).to include("Welcome to RecipeBox!")`